modules/sdk-core/src/bitgo/utils/tss/ecdsa/index.ts

```diff
@@ -0,0 +1,2 @@
+export { EcdsaUtils } from './ecdsa';
+export * as EcdsaUtilsTypes from './types';
```

Comment:
can we just 
```suggestion
export * from './ecdsa';
export * from './types';
```
and ditch the extra export in the [parent index](https://github.com/BitGo/BitGoJS/pull/2416/files#diff-94348c1eaa8e1cee2a08b2b4a9ea3fdc5297c7a193113cd26e765f33bd009d2aR4)